trustd/src/main.rs

```diff
@@ -60,16 +64,28 @@ async fn pm_mode() -> anyhow::Result<ExitCode> {
         "api",
         #[cfg(feature = "garage-door")]
         "--embedded-oidc",
+        "--sample-data",
+        "--db-port",
+        &postgres.settings().port.to_string(),
+    ]);
+
+    let importer = Trustd::parse_from([
```

Comment:
Wouldn't it be simpler to add `--with-importer` to the `api` var and omit the `importer` var?